packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -254,26 +311,61 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
           key: `${option.label}-${index}`,
           isSelected,
           onClick: () => {
-            !option.disabled && onChange?.(option.value);
-            close();
+            !option.disabled &&
+              onChange?.(
+                option.value,
+                Array.isArray(value) && value.includes(option.value)
+                  ? 'Removed'
+                  : 'Added',
```

Comment:
amm.. you want me to pass `CustomEvent` to the `onChange` handler or create new ones separate for addition and removal?
Cause if its just a type of the argument in function, whats the advantage we get versus we just have a custom type in there..?
But also not sure if we want to dispatch separate events and kinda change the existing API

Reply:
maybe we could keep the old API for single select and pass CustomEvent for multi select? but i'm not sure how it would work with react events so maybe not 🤔

one advantage we get is being able to `dispatchEvent` in tests for example

Reply:
But I want to `dispatchEvent` in `Select`, not in tests :?
hmm.. not sure. I only see that maybe instead of string, we could pass an object so in future we could add more data, if needed?